### PR TITLE
New language en-NZ

### PR DESCRIPTION
**Pull request by Gnito**
Added new language: English (New Zealand)

### CODE REVIEW

File: app/models/listing.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@
 #  delta                           :boolean          default(TRUE), not null
 #  open                            :boolean          default(TRUE)
 #  share_type_old                  :string(255)
+#  privacy                         :string(255)      default("private")
```
Comment (rap1ds) [inline app/models/listing.rb]: This is not related to this PR. Should be removed.

**Comment by rap1ds** [comment]
Good to go :+1:  There are some stuff in this PR that are unrelated, so a good practise in general is not to include any unrelated stuff to PR, but it doesn't matter that much. This can be merged.

**Comment by Gnito** [comment]
Yeah, there was only 1 line in this whole PR that was worth a PR process (addition to available locales). So, I decided to add those generated privacy changes into this one too - along with wti pull stuff that normally goes to master directly. Maybe those generated stuff should be but to master directly too? (if they are not added on their original branch)
